example/StateSquidSample.Android/MainActivity.cs

```diff
@@ -6,6 +6,7 @@
 using Android.Widget;
 using Android.OS;
 using StateSquidSample.Droid;
+using EasyLoadingSample.Droid;
```

Comment:
This using should be unneccesary, probably should change the namespace of whatever is needing it. EasyLoading is the old name of the package, so if that namespace is still in use we should edit it there.